### PR TITLE
Linux SSL instructions and other doc tweaks.

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-johnly** [comment]
Addresses https://github.com/blackbaud/skyux2/issues/1281. I sent Matt a screenshot of the new SSL instructions to get his feedback.

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html [hunk 1]
```diff
@@ -12,11 +12,11 @@
...
-        <li><p>Under the Keychains list on the left, select "login."</p></li>
-        <li><p>Under the Category list, select "Certificates."</p></li>
+        <li><p>Under the Keychains list on the left, select <strong>login</strong>.</p></li>
+        <li><p>Under the Category list, select <strong>Certificates</strong>.</p></li>
         <li><p>Drag <stache-code>skyux-ca.crt</stache-code> into the list of certificates.</p></li>
-        <li><p>Double-click the new "SKY UX" item in the list to open the certificate's info window.</p></li>
-        <li><p>Expand the Trust section near the top of the info window, and select the "Always Trust" option for "Secure Sockets Layer (SSL)."</p></li>
+        <li><p>Double-click the new <strong>SKY UX</strong> item in the list to open the certificate's info window.</p></li>
+        <li><p>Expand the Trust section near the top of the info window, and select the <strong>Always Trust</strong> option for Secure Sockets Layer (SSL).</p></li>
         <li><p>Close the info window.</p></li>
```
Comment (Blackbaud-TerryHelems) [inline src/app/learn/get-started/prereqs/ssl-certificate/index.html]: We don't want `SSL` emphasized?

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html [hunk 2]
```diff
@@ -25,47 +25,57 @@
...
+        <li><p>On the Certificate Manager, select <strong>OK</strong>.</p></li>
     </ol>
 
+    <stache-page-anchor>
+        Installation for Linux
+    </stache-page-anchor>
+    <ol>
+        <li><p>Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>.</p></li>
+        <li><p>Move the <stache-code>skyux-ca.crt</stache-code> to <stache-code>/usr/local/share/ca-certificates</stache-code>.</p></li>
+        <li><p>From the command prompt, run <stache-code>sudo update-ca-certificates</stache-code>.</p>
```
Comment (Blackbaud-TerryHelems) [inline src/app/learn/get-started/prereqs/ssl-certificate/index.html]: Does "command prompt" sound like Windows?  I would have though "terminal" for a Linux person, but maybe not?

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html [hunk 3]
```diff
@@ -25,47 +25,57 @@
...
+          <sky-alert alertType="info">When you use products that use their own certificate stores, you also need to add the certificate to those certificate stores. For example, follow the instructions above to install the certificate for Firefox, <a href="http://blogmines.com/blog/how-to-import-a-certificate-in-google-chrome/">import the certificate in Google Chrome</a>, or <a href="https://docs.microsoft.com/en-us/azure/java-add-certificate-ca-store">add the certificate to the Java certificates tore</a>.</sky-alert>
```
Comment (Blackbaud-BobbyEarl) [inline src/app/learn/get-started/prereqs/ssl-certificate/index.html]: Typo in the very last word.  "tore"

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/learn/get-started/prereqs/ssl-certificate/index.html]: I'll change it to "terminal."

File: src/app/learn/get-started/prereqs/ssl-certificate/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/learn/get-started/prereqs/ssl-certificate/index.html]: I'll emphasize it.

**Comment by blackbaud-johnly** [comment]
I addressed the 2 items @Blackbaud-TerryHelems pointed out and fixed the "tore" typo that @Blackbaud-BobbyEarl caught. Thanks for the feedback. ... I'll hold off on merging for just a little while to see if Matt has any additional feedback.

**Pull request merged**